src/extension/css/pom.xml

```diff
@@ -51,6 +51,23 @@
       <artifactId>easymockclassextension</artifactId>
       <scope>test</scope>
     </dependency>
+        <dependency>
+      <groupId>org.geoserver</groupId>
+      <artifactId>gs-rest</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Did not find any for CSS, only for ysld and was only doing writes. Maybe an uncommitted bit?